docs/pages/index.mdx

```diff
@@ -6,22 +6,32 @@ h1: Introduction
 
 ## What is Teleport
 
-Teleport is a Certificate Authority and an Access Plane for your infrastructure.
-With Teleport you can:
+Teleport is a Certificate Authority and an Access Plane for your infrastructure. With Teleport you can:
 
 - Set up single sign-on and have one place to access your SSH servers, Kubernetes, Databases, and Web Apps.
 - Use your favorite programming language to define access policies to your infrastructure.
 - Share and record interactive sessions across all environments.
 
-## What Makes Teleport Different
-
-- Teleport replaces legacy keys and shared secrets with short-lived X.509 and SSH certificates
-  for services and users.
-- It proxies and inspects SSH, Kubernetes, Web, and Database protocols.
-  For example for SSH, it controls the session from the start
-  and captures a session recording and in-kernel system calls using BPF.
-- It removes a need for VPN and can connect multiple regions and organizations
-  in a decentralized network using mutual TLS and SSH tunnels.
+<TileSet>
+  <Tile icon="bolt" title="Server Access" href="./getting-started.mdx">
+    Single Sign-On, short lived certificates and audit for SSH servers.
```

Comment:
This is great and a much better improvement overall!

* Nit: `Single Sign-On, short-lived certificates, and audit for SSH servers.`